docs/content/integrations/deltalake/reference.mdx

```diff
@@ -0,0 +1,416 @@
+def iris_dataset() -> pd.DataFrame:
+    return pd.read_csv(
+        "https://docs.dagster.io/assets/iris.csv",
+        names=[
+            "sepal_length_cm",
+            "sepal_width_cm",
+            "petal_length_cm",
+            "petal_width_cm",
+            "species",
+        ],
+    )
+```
+
+In this example, the `iris_dataset` asset will be stored in the `IRIS` schema, and the `daffodil_dataset` asset will be found in the `DAFFODIL` schema.
+
+<Note>
+  The two options for specifying schema are mutually exclusive. If you provide{" "}
+  <code>schema</code> configuration to the I/O manager, you cannot also provide
+  it via the asset key and vice versa. If no <code>schema</code> is provided,
+  either from configuration or asset keys, the default schema{" "}
+  <code>public</code> will be used.
+</Note>
+
+---
+
+## Using the Delta Lake I/O manager with other I/O managers
+
+You may have assets that you don't want to store in Delta Lake. You can provide an I/O manager to each asset using the `io_manager_key` parameter in the <PyObject object="asset" decorator /> decorator:
+
+```python file=/integrations/deltalake/multiple_io_managers.py startafter=start_example endbefore=end_example
+import pandas as pd
+from dagster_aws.s3.io_manager import s3_pickle_io_manager
+from dagster_deltalake import LocalConfig
+from dagster_deltalake_pandas import DeltaLakePandasIOManager
+
+from dagster import Definitions, asset
+
+
+@asset(io_manager_key="warehouse_io_manager")
+def iris_dataset() -> pd.DataFrame:
+    return pd.read_csv(
+        "https://docs.dagster.io/assets/iris.csv",
+        names=[
+            "sepal_length_cm",
+            "sepal_width_cm",
+            "petal_length_cm",
+            "petal_width_cm",
+            "species",
+        ],
+    )
+
+
+@asset(io_manager_key="blob_io_manager")
+def iris_plots(iris_dataset):
+    # plot_data is a function we've defined somewhere else
+    # that plots the data in a DataFrame
+    return plot_data(iris_dataset)
+
+
+defs = Definitions(
+    assets=[iris_dataset, iris_plots],
+    resources={
+        "warehouse_io_manager": DeltaLakePandasIOManager(
+            root_uri="path/to/deltalalke",
+            storage_options=LocalConfig(),
+            schema="iris",
+        ),
+        "blob_io_manager": s3_pickle_io_manager,
+    },
+)
+```
+
+In this example:
+
+- The `iris_dataset` asset uses the I/O manager bound to the key `warehouse_io_manager` and `iris_plots` uses the I/O manager bound to the key `blob_io_manager`
+- In the <PyObject object="Definitions" /> object, we supply the I/O managers for those keys
+- When the assets are materialized, the `iris_dataset` will be stored in Delta Lake, and `iris_plots` will be saved in Amazon S3
+
+---
+
+## Storing and loading PyArrow tables or Polars DataFrames in Delta Lake
+
+The Delta Lake I/O manager also supports storing and loading PyArrow and Polars DataFrames.
+
+<TabGroup>
+<TabItem name="PyArrow Tables">
```

Comment:
I think there's supposed to be a Polars example here too, but there's only one tab at the moment